### DISCUSSION TITLE
Trigger button click on Enter

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1085] [Ideas] [answered]
It'd be nice to have the ability to trigger a `primary` button click on **Enter**. I suppose this may get tricky, as a card/page may have more than one (primary) button. But this seems like a useful option to have, and something that may have been already considered. If it's already available / answered, I apologize.

**Comment by mturoci** [comment, marked as answer]
See https://github.com/h2oai/wave/issues/348
